docs/sftt/sftt-api-spec.md

```diff
@@ -1147,13 +1174,117 @@ Reservations imported successfully.
 
 If the request was malformed.
 
-##### `401 Unauthorized`
+##### `401 UNAUTHORIZED`
 
 If the user is not a super admin.
 
-### Import Trees
+### Import Sites
+
+`POST api/v1/protected/import/sites`
+
+Used to import sites into the database. See below for a description of each ambiguous field.
```

Comment:
The descriptions are helpful, if there's a specific format we're expecting (as opposed to just qualitative descriptions) we should specify that too. Status, confidence, coverage, and pruning specifically seem like fields that might have a specific format

Reply:
The format is sort of undecided... It might be set in the future but as of right now SFTT might want to update some of the options so it feels a little soon to "hard code" them into the back end already. Would prefer to see what Brandon will come up with for designs